var/spack/repos/builtin/packages/py-pyogrio/package.py

```diff
@@ -19,6 +19,8 @@ class PyPyogrio(PythonPackage):
     version("0.9.0", sha256="6a6fa2e8cf95b3d4a7c0fac48bce6e5037579e28d3eb33b53349d6e11f15e5a8")
 
     depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-wheel", type="build")
```

Comment:
FYI. This one is inherited from `PythonPackage`.

Reply:
I thought so, too, but when I updated to spack develop as of three weeks ago, the build failed due to a missing wheel dependency.